### PR TITLE
feat: add api key for ethgasstation

### PR DESCRIPTION
**Pull request by AdamDowson**
added api key for ethgasstation

### CODE REVIEW

File: src/app/util/gas.service.ts [hunk 1]
```diff
@@ -5,7 +5,8 @@ import { Subject } from 'rxjs';
 
 @Injectable()
 export class GasService {
-  private gasStationEndpoint = 'https://ethgasstation.info/json/ethgasAPI.json';
+  private gasStationEndpoint =
+    'https://ethgasstation.info/json/ethgasAPI.json?api-key=a517fc6d479ea48e5ecd60f1d298bf470f9d9f3615db61264718a10ab9e7';
```
Comment (rittme) [inline src/app/util/gas.service.ts:9]: Is it ok to have this key public?

Comment (AdamDowson) [reply src/app/util/gas.service.ts:9]: It's the only way for now, I have whitelisted app.request.network and baguette-app.request.network on DeFi pulse

Comment (benjlevesque) [reply src/app/util/gas.service.ts:9]: apparently they have a URL whitelist, so it should be fine?

Comment (AdamDowson) [reply src/app/util/gas.service.ts:9]: Yep, already added the URL whitelist - not perfect but fine for V1.

**Pull request merged**